### PR TITLE
Update NHS homepage template

### PR DESCRIPTION
**Pull request by frankieroberto**
A quick update to bring this in line with the current NHS homepage:

* Removed Coronavirus banner
* Change Coronavirus section to Women's Health
* Remove section "Help us help you get the treatment you need"
* Update links in NHS services
* Remove Health and care video library

### CODE REVIEW

File: docs/views/templates/nhsuk-homepage.html [hunk 1]
```diff
@@ -96,24 +96,6 @@
 </header>
 {% endblock %}
 
-{% block beforeContent %}
-<!-- Coronavirus (COVID-19) banner -->
```
Comment (edwardhorsford) [inline docs/views/templates/nhsuk-homepage.html:100]: Remove just the content, not the block?

File: docs/views/templates/nhsuk-homepage.html [hunk 2]
```diff
@@ -189,31 +171,22 @@ <h2>Medicines A to Z</h2>
 </section>
 
 
-<!-- Coronavirus content -->
+<!-- Women's health content -->
```
Comment (edwardhorsford) [inline docs/views/templates/nhsuk-homepage.html:174]:
```suggestion
<!-- Women’s health content -->
```

File: docs/views/templates/nhsuk-homepage.html [hunk 3]
```diff
@@ -189,31 +171,22 @@ <h2>Medicines A to Z</h2>
...
-        <div class="nhsuk-action-link">
-          <a class="nhsuk-action-link__link" href="" >
-            <svg class="nhsuk-icon nhsuk-icon__arrow-right-circle" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
-              <path d="M0 0h24v24H0z" fill="none"></path>
-              <path d="M12 2a10 10 0 0 0-9.95 9h11.64L9.74 7.05a1 1 0 0 1 1.41-1.41l5.66 5.65a1 1 0 0 1 0 1.42l-5.66 5.65a1 1 0 0 1-1.41 0 1 1 0 0 1 0-1.41L13.69 13H2.05A10 10 0 1 0 12 2z"></path>
-            </svg>
-            <span class="nhsuk-action-link__text">Find out about COVID-19</span>
-          </a>
+          <h2>Women's health</h2>
+          <p class="app-section__description">Find information about women's health, including periods, contraception, conditions and screening</p>
```
Comment (edwardhorsford) [inline docs/views/templates/nhsuk-homepage.html:181]:
```suggestion
          <h2>Women’s health</h2>
          <p class="app-section__description">Find information about women’s health, including periods, contraception, conditions and screening</p>
```

Comment (edwardhorsford) [inline docs/views/templates/nhsuk-homepage.html:181]: Should this have a full stop?

File: docs/views/templates/nhsuk-homepage.html [hunk 4]
```diff
@@ -189,31 +171,22 @@ <h2>Medicines A to Z</h2>
...
         </div>
         <div class="nhsuk-action-link">
           <a class="nhsuk-action-link__link" href="" >
             <svg class="nhsuk-icon nhsuk-icon__arrow-right-circle" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
               <path d="M0 0h24v24H0z" fill="none"></path>
               <path d="M12 2a10 10 0 0 0-9.95 9h11.64L9.74 7.05a1 1 0 0 1 1.41-1.41l5.66 5.65a1 1 0 0 1 0 1.42l-5.66 5.65a1 1 0 0 1-1.41 0 1 1 0 0 1 0-1.41L13.69 13H2.05A10 10 0 1 0 12 2z"></path>
             </svg>
-            <span class="nhsuk-action-link__text">Find out about COVID-19 vaccination</span>
+            <span class="nhsuk-action-link__text">Go to women's health</span>
```
Comment (edwardhorsford) [inline docs/views/templates/nhsuk-homepage.html:189]:
```suggestion
            <span class="nhsuk-action-link__text">Go to women’s health</span>
```

**Review by edwardhorsford — APPROVED** [review]
Tested locally, looks good. A few suggestions though.

File: docs/views/templates/nhsuk-homepage.html (hunk 3 above)
Comment (frankieroberto) [reply docs/views/templates/nhsuk-homepage.html:181]: Doesn't on the NHS homepage... 🤷‍♂️

File: docs/views/templates/nhsuk-homepage.html (hunk 4 above)
Comment (frankieroberto) [reply docs/views/templates/nhsuk-homepage.html:189]: copied this from upstream on NHS homepage

File: docs/views/templates/nhsuk-homepage.html (hunk 1 above)
Comment (frankieroberto) [reply docs/views/templates/nhsuk-homepage.html:100]: not sure there's any need to keep the block if it's empty? (It's still defined on the template this template extends)

**Pull request merged**